Added tests for README (#100)
apple/swift-format: README.md

that are compatible with the latest release branch will be cherry-picked into
that branch.

> You can test if the compilation was successful with:

Reviewer: ```suggestion
To test that the formatter was built successfully and is compatible with your Swift toolchain, you can run the following command:
```

Author: Done

```

> swift test

Reviewer: ```suggestion
swift test --parallel
```

Author: Done

```

> 

Reviewer: ```suggestion

We recommend using the `--parallel` flag to speed up the test run since there are a large number of tests.
```

Author: Done
